src/components/Modal/Modal.unit.spec.jsx

```diff
@@ -70,7 +84,22 @@ describe('<Modal/>', () => {
       global.document.body.removeChild(root);
     });
 
-    it('should start on the first focusable element by default', () => {
+    it('should open with focus assigned to the close button', () => {
+      const tree = mount(
+        <Modal id="modal" title="Programmatic focus" visible onClose={() => {}}>
+          <button id="first-button">First button</button>
+          <button id="second-button">Second button</button>
+        </Modal>,
+        { attachTo: root },
+      );
+
+      expect(global.document.activeElement.className).to.equal(
+        'va-modal-close',
+      );
+      tree.unmount();
+    });
+
+    it('should open with focus assigned to the first button when the close button is turned off', () => {
```

Comment:
It focuses on the first focusable item inside the modal. For most modals, this is the close button just because it appears in the top right corner and ahead of any other buttons.